### ISSUE TITLE
[suggestion] with --multi add option to pre-select first n options

### ISSUE THREAD

**Issue by loa-in-corporate** [junegunn/fzf#1478]
- Category
    - [x] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [x] Linux
    - [x] Mac OS X
    - [x] Windows
    - [x] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish

The feature, while simple, allows one to save state between runs of fzf, simplifying a lot of use cases.

The feature would add a command-line parameter "--multi-preselect <n>" which would start fzf with top (or bottom) n entries selected (in input order).

Use case (bash):
> function tests_rerun() {
>     # .tests.profile contains saved set of tests to run
>     ./gradlew $(cat .tests.profile | tr -s '\r\n' '  ')
> }
> function tests_select() {
>     # .tests.profile contains saved set of tests to run
>     # merge saved set with set of all known tests, preselect all from saved set
>     cat <(cat .tests.profile) <(project-list --tests --all) | fzf --multi --multi-preselect $(wc -l .tests.profile) | sort -u > new.tests.profile
>     # save as new set
>     mv new.tests.profile .tests.profile
> }
> function lazy_fingers() {
>     tests_select; tests_rerun; tests_rerun; tests_select; tests_rerun
> }

In other words it would allow one to add or modify selection instead of selecting everything each time from scratch. This is currently impossible to do in fzf if I'm not mistaken.

**Comment by loa-in-corporate** [comment]
I can improve my description if anyone would bother me to.

**Comment by chthor** [comment]
+1 

This would be super handy. What I could actually use is the ability to pre-select arbitrary options, presumably by sending in a list of strings or indexes.

**Comment by junegunn** [comment]
```sh
seq 100 | fzf --multi --sync --bind 'result:transform:for _ in $(seq 5); do echo -n "+select+up"; done'
seq 100 | fzf --multi --sync --bind 'result:last+transform:for _ in $(seq 10); do echo -n "+select+down"; done'
```
(Use the latest 0.54.0 for the best result)
